dbdialog/src/main/java/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -764,7 +764,15 @@ protected void getInfo( DatabaseMeta meta ) {
     if ( meta.isUsingConnectionPool() ) {
       if ( poolSizeBox != null ) {
         try {
-          int initialPoolSize = Integer.parseInt( poolSizeBox.getValue() );
+          int initialPoolSize;
```

Comment:
I think you're looking for databaseMeta.environmentSubstitute here - you shouldn't be picking apart the variable like this.

Reply:
@mbatchelor Now I believe is ready to be merged.